src/util/is-production.js

```diff
@@ -0,0 +1,7 @@
+let environment = process && process.env ? process.env.NODE_ENV : null;
```

Comment:
Will uglification still remove the dead code if we're using this utility? I'm not sure t will. I think it explicitly needs to see `process.env.NODE_ENV === 'development'` to remove it.